frontend/src/components/filters/FiltersIncluded.js

```diff
@@ -202,26 +188,7 @@ class FiltersNotFrequent extends PureComponent {
   }
 }
 
-/**
- * @typedef {object} Props Component props
```

Comment:
I think I've asked in another file, but why are we removing those ?

Reply:
It's just because the format of the Proptypes is self-explanatory of what the param is and what type it has. I am for having these comments but only when is needed. In this case this would just add extra lines to the component that deliver the same information that exist in the Proptypes and that I found to be redundant.